lib/rollbar_helper/version.rb

```diff
@@ -0,0 +1,3 @@
+module RollbarHelper
+  VERSION = '0.2.1'
```

Comment:
I think this should up the minor instead though

Reply:
You’re right, I wasn’t sure if you said minor or patch this morning but I thought minor made more sense if this is following semver

Reply:
Fixed

Reply:
👍